cli/flox-rust-sdk/src/models/search.rs

```diff
@@ -40,3 +40,17 @@ pub struct SearchResult {
     /// The package description
     pub description: Option<String>,
 }
+
+/// Details about a single build of a package
```

Comment:
In retrospect this could have been `type PackageBuild = SearchResult;` but it goes away in a commit or two anyway.